index.src.html

```diff
@@ -896,19 +907,25 @@ <h4 id="fetching">Fetching</h4>
                     blocked as mixed content
                   </a> returns **allowed**, return null.
 
+              1.  If <a http-header>`Private-Network-Access-ID`</a> or
+                  <a http-header>`Private-Network-Access-Name`</a> is null or
+                  empty, let |targetId| be |request|'s [=request/target IP
+                  address space=]. Store the permission as an ephemeral
+                  permission, then return null.
+  
               1.  Let |targetId| be the result of [=extracting header list
                   values=] given
                   "<a http-header>`Private-Network-Access-ID`</a>" and
                   |response|'s [=response/header list=].
 
-              1.  if |targetId| is null or empty, return a [=network error=].
+              1.  if |targetId| is invalid, return a [=network error=].
```

Comment:
I think we should just put the requirement in the algorithm.   Something like:

```suggestion
              1.  if |targetId| is not a string of 6 hexadecimal bytes separated by colons, return a [=network error=].
```